aspnetcore/security/authentication/identity-enable-qrcodes.md

```diff
@@ -14,7 +14,7 @@ uid: security/authentication/identity-enable-qrcodes
 
 # Enabling QR Code generation for authenticator apps in ASP.NET Core
 
-Note: This topic applies to ASP.NET Core 2.x
+Note: This topic applies to ASP.NET Core 2.x and Razor Pages.
```

Comment:
It may be better to say "with Razor Pages" instead of "and Razor Pages". Some may interpret "and Razor Pages" as "it works with all of ASP.NET Core 2.x including Razor Pages.